database/sql/src/main/scala/cromwell/database/slick/tables/WorkflowMetadataSummaryEntryComponent.scala

```diff
@@ -266,10 +282,11 @@ trait WorkflowMetadataSummaryEntryComponent {
     val orderByAddendum = sql""" ORDER BY WORKFLOW_METADATA_SUMMARY_ENTRY_ID DESC
                                | """.stripMargin
 
-
     // NB you can preview the prepared statement created here by using, for example: println(result.statements.head)
 
     concatNel((NonEmptyList.of(mainQuery) :+ orderByAddendum) ++ paginationAddendum)
-      .as[WorkflowMetadataSummaryEntry](rconv = GetResult(r => WorkflowMetadataSummaryEntry(r.<<, r.<<, r.<<, r.<<, r.<<, r.<<, r.<<)))
+      .as[WorkflowMetadataSummaryEntry](rconv = GetResult { r =>
+      WorkflowMetadataSummaryEntry(r.<<, r.<<, r.<<, r.<<, r.<<, r.<<, r.<<, r.<<, r.<<)
```

Comment:
What does `r.<<` do?

Reply:
- [Documentation](http://slick.lightbend.com/doc/3.2.3/sql.html#result-sets)
- Implementation
  - PositionedResult [summoner](https://github.com/slick/slick/blob/v3.2.3/slick/src/main/scala/slick/jdbc/PositionedResult.scala#L31) for GetResult[T] instances
  - Example GetResult[[Boolean]](https://github.com/slick/slick/blob/v3.2.3/slick/src/main/scala/slick/jdbc/GetResult.fm#L21)
  - Example: PositionedResult.[nextBoolean()](https://github.com/slick/slick/blob/v3.2.3/slick/src/main/scala/slick/jdbc/PositionedResult.scala#L34)

`r` is (a wrapper around) the mutable JDBC result set. Usually one advances JDBC the result set from one row to the next. On any single row, assuming the SQL has requested a fixed, ordered list of column, the columns are accessed one at a time from `0` to `num_columns-1`.

This wrapper allows one to instead use the `<<` operator to "get the next column value and then increment the internal column index". So each time one calls `<<` the internal pointer is also moving to the next column, ready for access by the next call.

With parameter names, types, and implicits this would look like:
```scala
WorkflowMetadataSummaryEntry(
  workflowExecutionUuid          = r.<<[String](GetString),
  workflowName                   = r.<<[Option[String]](GetStringOption),
  workflowStatus                 = r.<<[Option[String]](GetStringOption),
  startTimestamp                 = r.<<[Option[Timestamp]](GetTimestampOption),
  endTimestamp                   = r.<<[Option[Timestamp]](GetTimestampOption),
  submissionTimestamp            = r.<<[Option[Timestamp]](GetTimestampOption),
  workflowMetadataSummaryEntryId = r.<<[Option[Long]](GetLongOption)
)
```

Reply:
Thank you! :)